acceptance-tests/tests/simple-permissioning-smart-contract/test/test-node-permissioning.js

```diff
@@ -19,12 +19,9 @@ contract('Permissioning: Nodes', () => {
   });
 
   it('Should compute key', async () => {
-    let key1 = await proxy.computeKey(node1High, node1Low, node1Host, node1Port);
-    let key2 = await proxy.computeKey(node1High, node1Low, node1Host, node1Port);
+    let key1 = await proxy.computeKey(node1High, node1Low, node1Host);
+    let key2 = await proxy.computeKey(node1High, node1Low, node1Host);
     assert.equal(key1, key2, "computed keys should be the same");
-
-    let key3 = await proxy.computeKey(node1High, node1Low, node1Host, node2Port);
-    assert(key3 != key2, "keys for different ports should be different");
```

Comment:
can we keep a test for generating different keys - eg change the host value instead of port

Reply:
Good idea.